src/qt/coincontroldialog.cpp

```diff
@@ -691,8 +690,7 @@ void CoinControlDialog::updateView()
     map<QString, vector<COutput> > mapCoins;
     model->listCoins(mapCoins);
 
-    BOOST_FOREACH(PAIRTYPE(QString, vector<COutput>) coins, mapCoins)
-    {
+    BOOST_FOREACH(const PAIRTYPE(QString, vector<COutput>) coins, mapCoins) {
```

Comment:
Do make it a reference as well:
```
 BOOST_FOREACH(const PAIRTYPE(QString, vector<COutput>) &coins, mapCoins) {
```
If a copy is made anyway, making it const has little benefit.